File: baekjoon/js/2438.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

const n = Number(buf);

const result = [];

for (let i = 1; i <= n; i++) {
  result.push("*".repeat(i));
}

console.log(result.join("\n"));

File: baekjoon/rust/2438.rs
use std::io::stdin;

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let n: i32 = buf.trim().parse().unwrap();

    let mut result = Vec::new();

    for i in 1..=n {
        result.push("*".repeat(i as usize));
    }

    print!("{}", result.join("\n"));
}
